### PR TITLE
SDA-2952 - Add "hidden" field to AddOn

### PR DESCRIPTION
**Pull request by mikenairn**
JIRA: https://issues.redhat.com/browse/SDA-2952

### CODE REVIEW

File: model/clusters_mgmt/v1/add_on_type.model [hunk 1]
```diff
@@ -34,6 +34,9 @@ class AddOn {
 	// Indicates if this add-on can be added to clusters.
 	Enabled Boolean
 
+	// Indicates if this add-on is hidden.
```
Comment (mikenairn) [inline model/clusters_mgmt/v1/add_on_type.model:37]: Maybe not the best description. :thinking:

Comment (vkareh) [reply model/clusters_mgmt/v1/add_on_type.model:37]: meh, looks fine to me

**Pull request merged**